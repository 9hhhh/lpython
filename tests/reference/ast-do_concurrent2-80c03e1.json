{
    "basename": "ast-do_concurrent2-80c03e1",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/do_concurrent2.f90",
    "infile_hash": "afe8788676d4b7be4b765236db5d9c60aa267325a08b714b35ba7eb6",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-do_concurrent2-80c03e1.stdout",
    "stdout_hash": "158cd2344e2ca8a1f4b6ed6c0983db3bd4a0936359b746c42f0d0824",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}