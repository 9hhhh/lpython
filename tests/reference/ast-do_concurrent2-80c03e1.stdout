(TranslationUnit [(Subroutine do_concurrent2 [(a) (b)] [] () [] [] [] [(Declaration (AttrType TypeReal [] ()) [(AttrIntent InOut)] [(a [(() () DimensionExpr)] [] ()) (b [(() () DimensionExpr)] [] ())]) (Declaration (AttrType TypeInteger [] ()) [] [(i [] [] ())]) (Declaration (AttrType TypeReal [] ()) [] [(x [] [] ())])] [(= 0 x (Real "1.0")) (DoConcurrentLoop 0 () [(ConcurrentControl i 1 10 ())] () [(ConcurrentShared [i]) (ConcurrentLocal [x]) (ConcurrentDefault)] [(If 0 () (> (FuncCallOrArray a [] [(() i () CodimensionExpr)] []) 0) [(= 0 x (FuncCallOrArray sqrt [] [(() (FuncCallOrArray a [] [(() i () CodimensionExpr)] []) () CodimensionExpr)] [])) (= 0 (FuncCallOrArray a [] [(() i () CodimensionExpr)] []) (- (FuncCallOrArray a [] [(() i () CodimensionExpr)] []) (** x 2)))] []) (= 0 (FuncCallOrArray b [] [(() i () CodimensionExpr)] []) (- (FuncCallOrArray b [] [(() i () CodimensionExpr)] []) (FuncCallOrArray a [] [(() i () CodimensionExpr)] [])))]) (Print 0 () [x])] [])])
